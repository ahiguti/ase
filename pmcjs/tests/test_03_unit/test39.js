
print("A");
try {
  Perl.Eval("$foo(3)");
} catch (e) {
  print(e);
}
print("B");
