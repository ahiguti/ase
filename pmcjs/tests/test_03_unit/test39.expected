A
(
 etype: RuntimeException,
 error: PerlException,
 func: pmc_eval_internal,
 description: syntax error at (eval 3) line 1, near "$foo(" ,
 at: 
  func=[] file=test39.js line=4
)
B
